config.yaml

```diff
@@ -37,3 +37,8 @@ options:
     description: |
       Comma-separated list of Resources to override the default set of Resources to monitor.
     default: ""
+  scrape-interval:
```

Comment:
this allows for flexible configuration of how often the endpoints are scraped by prometheus